### DISCUSSION TITLE
Support Middle Mouse Click and Shift + Click in Taskbar when using tabs

### DISCUSSION THREAD

**Discussion by chris400** [sumatrapdfreader/sumatrapdf#3384] [Ideas]
Most applications nowadays support Middle Mouse Click (aka Wheel Click) or Shift + Click on Taskbar to open up a new window. SumatraPDF supports this functionality only, when tabs are disabled. I would like to suggest to support this behavior when using tabs also.

**Comment by kjk** [comment]
Please open an issue so it's not forgotten.

**Comment by chris400** [comment]
https://github.com/sumatrapdfreader/sumatrapdf/issues/3386
